docs/development/test-vectors.rst

```diff
@@ -409,6 +409,10 @@ Custom X.509 Certificate Revocation List Vectors
 * ``crl_delta_crl_indicator.pem`` - Contains a CRL with the
   ``DeltaCRLIndicator`` extension.
 
+X.509 OCSP Test Vectors
+~~~~~~~~~~~~~~~~~~~~~~~
+* ``x509/ocsp/resp-sha256.der`` - An OCSP response with a SHA256 signature.
```

Comment:
Please mention that it's an OCSP response for cryptography.io

Reply:
Is that a useful piece of information? The cert it's linked to will have been rotated in the next 90 days.